test/data/collection.cpp

```diff
@@ -502,19 +508,21 @@ BOOST_AUTO_TEST_CASE(collection__reverse__const__reversed)
 
 BOOST_AUTO_TEST_CASE(collection__sort__empty__empty)
 {
-    BOOST_REQUIRE(sort(data_chunk{}).empty());
+    data_chunk data{};
+    BOOST_REQUIRE(sort(data).empty());
 }
 
 BOOST_AUTO_TEST_CASE(collection__sort__single__unchanged)
 {
     const uint8_t expected = 42;
-    BOOST_REQUIRE_EQUAL(sort(data_chunk{ expected }).front(), expected);
+    data_chunk data{ expected };
+    BOOST_REQUIRE_EQUAL(sort(data).front(), expected);
 }
 
 BOOST_AUTO_TEST_CASE(collection__sort__const_unsorted__sorted)
 {
     const data_chunk set{ 2, 0, 0, 8, 6, 4 };
-    const data_chunk expected{ 0, 0, 2, 4, 6, 8 };
+    data_chunk expected{ 0, 0, 2, 4, 6, 8 };
```

Comment:
FYI, I recently renamed collection copy overloads to _copy to avoid ambiguity when a copy is explicitly desired.

Reply:
The return would just be `Collection`